Assets/Scripts/Models/Prototypes/PrototypeManager.cs

```diff
@@ -28,6 +29,12 @@ public static BasePrototypes<Job> FurnitureJob
         protected set;
     }
 
+    public static BasePrototypes<Job> FurnitureJobDeconstruct
```

Comment:
This can be one line
`public static BasePrototypes<Job> FurnitureJobDeconstruct { get; private set; }`
and why protected? Are you going to assign this property from derived class?